src/mbedtls_context.cpp

```diff
@@ -673,32 +674,46 @@ namespace sockpp {
     }
 
 
-    // callback from mbedTLS cert validation (see above)
+    // Callback from mbedTLS cert validation (see above)
+    //
+    // When a pinned cert is specified, the verify_callback will compare the pinned cert with
+    // each cert in the chain. If one of the certs in the chain matches with the pinned cert,
+    // the server servers are trusted.
```

Comment:
"is trusted" or "are trusted" ?

Reply:
There is a typo as well. Will rewrite the sentence.